app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<p>
+  Hello, <%= @org.name %>
+</p>
+
+<p>
+<% if @organization_campaign %>
+  Your current campaign is <%= @organization_campaign.campaign.name %>
+
+  <table>
+    <tr>
+      <th>Assigned Recipients</th>
+      <th>Matched</th>
+      <th>Percent</th>
+    </tr>
+    <tr>
+      <td><%= @organization_campaign.assigned %></td>
+      <td><%= @organization_campaign.matched %></td>
+      <td><strong><%= @organization_campaign.matched_pct %></strong></td>
+    </tr>
+  </table>
+
+<% else %>
+  You are not joined to any campaign at the moment.  Perhaps you should join one!
+<% end %>
+</p>
+
+<%= form_for @organization_campaign_new, 
+    url: {action: :create, controller: :organization_campaigns} do |f| %>
+  Join campaign: 
+  <%= f.hidden_field :organization_id %>
+  <%= f.collection_select(:campaign_id, @joinable_campaigns,:id, :name) %>
+  <%= f.submit "Join" %>
+<% end %>
+
+<% if @joined_campaigns %>
```

Comment:
Yeah, it's just to view older campaigns.  It was pretty easy to add this as a user story because the Current Campaign needs to be passed to pretty much every page in the Org Admin.  (We do it in session[:current_campaign].)  To make it less confusing, I now hide the box if the org belongs to only one campaign.